app/jobs/clean_stalled_uphold_codes_and_access_parameters_job.rb

```diff
@@ -0,0 +1,30 @@
+class CleanStalledUpholdCodesAndAccessParametersJob < ApplicationJob
+  queue_as :scheduler
+
+  def perform
+    # clear uphold codes sitting for over 5 minutes   
+    publishers = Publisher.has_stalled_uphold_code
+    n = 0
+    publishers.each do |publisher|
+      raise if publisher.uphold_status != :code_acquired
+      publisher.uphold_code = nil
+      publisher.uphold_updated_at = Time.now
+      publisher.save!
+      n += 1
+      Rails.logger.info("Cleaned stalled uphold code for #{publisher.brave_publisher_id}.")
```

Comment:
- We should log the publisher.id
- Also I think we should send events to Sentry ([Raven.capture_message](https://docs.sentry.io/clients/ruby/usage/#optional-attributes)) to become aware of the extent of this issue; then we'll review in the medium future.